src/visualization/utils/useVisDomainSettings.ts

```diff
@@ -242,10 +242,11 @@ export const useZoomRequeryXDomainSettings = (args: ZoomRequeryArgs) => {
         setRequeryStatus(RemoteDataState.Loading)
         runQuery(orgId, query, extern).promise.then(
           (result: RunQueryResult) => {
-            if (result.type === 'SUCCESS') {
+            if (result?.type === 'SUCCESS') {
               setRequeryStatus(RemoteDataState.Done)
-              const parsed = fromFlux(result.csv)
-              setResult(parsed)
+              if (result?.csv?.trim().length > 0) {
```

Comment:
I think this line is fine, but I want to make sure it doesn't open this expression up to something like `undefined.trim()`.

Reply:
No, if any part of the chain is `undefined` then it becomes `if (undefined > 0)` which is false

Reply:
Here is a quick proof from the console

<img width="458" alt="Screen Shot 2022-09-02 at 9 24 13 AM" src="https://user-images.githubusercontent.com/10736577/188197402-b2687010-bd0c-4187-8fce-3ec38d96696d.png">